src/pytkdocs/loader.py

```diff
@@ -415,6 +431,49 @@ def get_property_documentation(self, node: ObjectNode) -> Attribute:
             source=source,
         )
 
+    def get_pydantic_field_documentation(self, node: ObjectNode) -> Attribute:
+        """
+        Get the documentation for a PyDantic Field
+
+        Arguments:
+            node: The node representing the Field and its parents.
+
+        Return:
+            The documented attribute object.
+        """
+        prop: ModelField = node.obj
+        path = node.dotted_path
+        properties = ["field"]
+        if prop.required:
+            properties.append("required")
+
+        return Attribute(
+            name=node.name,
+            path=path,
+            file_path=node.file_path,
+            docstring=prop.field_info.description,
```

Comment:
So, we don't have to parse a triple-quoted docstring below the field declaration, good :+1: